spec/factories/purls.rb

```diff
@@ -30,6 +34,12 @@
       contents { [build(:resource, :document)] }
     end
 
+    trait :document_no_download do
+      type { 'document' }
+      title { 'Title of the object' }
+      contents { [build(:resource, :document_no_download)] }
+    end
+
     trait :was_seed do
```

Comment:
we need to ensure the public purls created for specs are marked as public and have a world downloadable file or else the PDF viewer will restrict viewing now